### PR TITLE
Remove separate ARM build

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
As https://github.com/jamulussoftware/jamulus/pull/2808 was merged, we now have a universal build making the need of separate ARM and Intel downloads for macOS unneeded
**Context: Fixes an issue? Related issues**
Related to: https://github.com/jamulussoftware/jamulus/pull/2808
**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Review

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by ann0see** [comment]
Probably needs a rebase on my side...

**Comment by pljones** [comment]
Could do with review by a Mac owner.  Preferably three Mac owners (legacy, post-legacy Intel, Apple Silicon).

**Comment by ann0see** [comment]
I think the binary is already tested OK. That's just a change on the website.

**Comment by pljones** [comment]
> I think the binary is already tested OK. That's just a change on the website.

Yes.  I'd like people who know what to look for to read it and see if it makes enough sense.

File: wiki/en/Installation-for-Macintosh.md [hunk 1]
```diff
@@ -13,22 +13,9 @@ Make sure you've already read the [Getting Started](Getting-Started) page.
...
-[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}
-
-For macOS Mojave (10.14 or lower) please [download the legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})
-
-**For macOS running on Apple Silicon:**
-
-[Download Jamulus (Apple Silicon)]({{ site.download_root_link }}{{ site.download_file_names.mac-arm }}){: .button}
-
+1. [Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\
+ **macOS Mojave (10.14) or lower:** [Download legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})\\
```
Comment (gilgongo) [inline wiki/en/Installation-for-Macintosh.md:17]: Is "universal build" standard vocabulary for Mac users? What happens if we just give a minimum OS version?

Comment (ann0see) [reply wiki/en/Installation-for-Macintosh.md:17]: Yes, I think so. image

Comment (gilgongo) [reply wiki/en/Installation-for-Macintosh.md:17]: OK well in that case it looks good to me.

**Review by gilgongo — APPROVED** [review]

**Pull request merged**